packages/knip/src/plugins/dependency-cruiser/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://github.com/sverweij/dependency-cruiser
+
+const title = 'dependency-cruiser';
+
+const enablers = ['dependency-cruiser'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const config = ['.dependency-cruiser.{js,cjs,mjs,json}'];
+
+const args = {
+  binaries: ['depcruise'],
```

Comment:
looks like they have many: https://github.com/sverweij/dependency-cruiser/blob/c48aca959c9c9c1b1068c6d85cd4ed368de10d27/package.json#L74-L80

any chance you could add those as well?

Reply:
Yes, added the relevant ones